testsetup/testsetup.go

```diff
@@ -73,7 +73,7 @@ func (c Command) BuildCommand(args ...string) *exec.Cmd {
 }
 
 var (
-	GoVersions = []string{"go1.12.14", "go1.13.5", "go1.14beta1"}
+	GoVersions = []string{"go1.12.14", "go1.13.5", "go1.14rc1"}
```

Comment:
Shouldn't we update to `go1.12.16` and `go1.13.7` while at it?

Reply:
Yes, we should. Nice spot :)